.github/workflows/editor-tests.yml

```
@@ -16,10 +16,13 @@ jobs:
     strategy:
       matrix:
         # os: [ubuntu-20.04, macos-13, windows-2019]
-        os: [ubuntu-20.04, macos-13]
+        os: [ubuntu-latest, macos-13]
       fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
+    - name: Install build dependencies
+      run: sudo apt-get update && sudo apt-get install -y libx11-dev libxkbfile-dev pkg-config libsecret-1-dev libncursesw5-dev libgdbm-dev libc6-dev libffi-dev
+    
```

Tiny oversight:

Since `apt-get install` is Linux-specific, and this is inside a two-OS matrix job definition (macOS and Linux), we should make sure this only runs on Linux. Code suggestion below:

```suggestion
    - name: Install build dependencies
      if: runner.os == 'Linux'
      run: sudo apt-get update && sudo apt-get install -y libx11-dev libxkbfile-dev pkg-config libsecret-1-dev libncursesw5-dev libgdbm-dev libc6-dev libffi-dev
    
```